src/commands/dev/index.ts

```diff
@@ -175,30 +175,34 @@ export class UpProcessManager {
     }
 
     const service_colors = new Map<string, chalk.Chalk>();
-    this.compose_process.stdout?.on('data', (data) => {
-      if (this.is_exiting) {
-        return;
-      }
+    const createHandleStream = (output_func: (message: string) => void) => {
```

Comment:
I also noticed that we don't pipe stderr here:
```ts
const compose_process = DockerComposeUtils.dockerCompose(compose_args,
      { stdout: 'pipe', stdin: 'ignore', detached: !this.is_windows });
```

Reply:
Oh I guess the default is pipe so maybe not a big deal, but confusing to read

Reply:
Fair I can make that more obvious so no one spends time going down some kind of wrong rabbit hole.